overlord/snapshotstate/export_test.go

```diff
@@ -32,19 +32,32 @@ import (
 )
 
 var (
-	NewSnapshotSetID          = newSnapshotSetID
-	AllActiveSnapNames        = allActiveSnapNames
-	SnapNamesInSnapshotSet    = snapNamesInSnapshotSet
-	CheckSnapshotTaskConflict = checkSnapshotTaskConflict
-	Filename                  = filename
-	DoSave                    = doSave
-	DoRestore                 = doRestore
-	UndoRestore               = undoRestore
-	CleanupRestore            = cleanupRestore
-	DoCheck                   = doCheck
-	DoForget                  = doForget
+	NewSnapshotSetID           = newSnapshotSetID
+	AllActiveSnapNames         = allActiveSnapNames
+	SnapSummariesInSnapshotSet = snapSummariesInSnapshotSet
+	CheckSnapshotTaskConflict  = checkSnapshotTaskConflict
+	Filename                   = filename
+	DoSave                     = doSave
+	DoRestore                  = doRestore
+	UndoRestore                = undoRestore
+	CleanupRestore             = cleanupRestore
+	DoCheck                    = doCheck
+	DoForget                   = doForget
 )
 
+func (shsums snapshotSnapSummaries) Asplode() []map[string]string {
```

Comment:
Hm, maybe `func (sss snapshotSnapSummaries) Summaries() []map[string]string {` ? The `shsums` reads a lot like hashsums to me and Asplode is a hard to remember :)

Reply:
sure